helper/deploy_pages.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -eu
+
+# setup variables
+access_token=$1
+user_name=$2
+user_mail=$3
+
+repo_uri="https://x-access-token:${access_token}@github.com/OSInside/kiwi.git"
+remote_name="origin"
+target_branch="gh-pages"
+
+# set git user and email
+git config user.name "${user_name}"
+git config user.email "${user_mail}"
+
+# build kiwi documentation suitable for github pages
+tox -e doc_travis
```

Comment:
I find it confusing... I understand we are not using travis to build docs, just the `travis-sphinx` python tool. We use it only to build the docs, not even to push them to giuthub as I understand this is done later on in this script.

To me all these references to travis are confusing, unless you really pay attention to the code it looks like docs are all build on travis. I'd say we should just keep `travis` literal for the `travis-sphinx` tool and name `tox` job something different than `doc_travis` and use a different temporary folder than `build_travis`.